lib/classes/Variables.js

```diff
@@ -822,6 +827,24 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    return BbPromise.try(() => {
+      if (_.isString(variableString)) {
+        const groups = variableString.match(this.strToBoolRefSyntax);
+        const param = groups[1].trim();
+        if (param === 'false') {
+          return false;
+        } else if (!param) {
```

Comment:
I think it's a dead condition, as regex doesn't allow empty (or just whitespace content)

So either we should remove that condition, or relax regex to allow empty (or just whitespace) strings

Reply:
Sounds good. I removed the condition.